Gemfile

```diff
@@ -2,7 +2,8 @@ source 'https://rubygems.org'
 
 gem 'rspec'
 gem "rack"
-gem "sinatra"
+gem "sinatra", "~> 2.0.0.rc2"
```

Comment:
Does this mean we now _only_ support Rack 2 or does the code still work w/ Rack 1? Can we be sure? 😄